sdks/python/apache_beam/testing/synthetic_pipeline.py

```diff
@@ -415,19 +418,24 @@ def get_range_tracker(self, start_position, stop_position):
       tracker = range_trackers.UnsplittableRangeTracker(tracker)
     return tracker
 
+  @staticmethod
+  def random_bytes(length):
+    """Return random bytes."""
+    return b''.join(
+        (struct.pack('B', random.getrandbits(8)) for _ in xrange(length)))
```

Comment:
Let's use range instead of xrange, and adjust the import accordingly. We try to use idiomatic py3 whenever possible in Beam.

Reply:
Alright. I was a bit concerned about performance of `range` on Python 2, but the parameter is almost always small so the impact is negligible.

Reply:
We can add `from builtins import range` to use py3 implementation of range on py2.